modules/structs/issue.go

```diff
@@ -110,6 +110,7 @@ type EditIssueOption struct {
 	// swagger:strfmt date-time
 	Deadline       *time.Time `json:"due_date"`
 	RemoveDeadline *bool      `json:"unset_due_date"`
+	ClosedStatus   int8       `json:"closed_status"`
```

Comment:
IMO since the `State` above is string "closed/open", for consistency, the "ClosedStatus" should also have its string representation.

Reply:
fbb1bdd, "ClosedStatus" accepts string value